### PR TITLE
Add support for Debian Bullseye repo

### PR DESCRIPTION
**Pull request by gbp**
Available since https://github.com/mysociety/sysadmin/issues/1507

### CODE REVIEW

**Review by sagepe — CHANGES_REQUESTED** [review]
Thanks for doing this @gbp - it had slipped my mind. Slightly annoyingly, [there's a change in Bullseye to the format of the security release line](https://wiki.debian.org/DebianBullseye#FAQ), so we'll need to cater for that.

File: bin/install-site.sh [hunk 1]
```diff
@@ -160,8 +166,8 @@ deb http://the.earth.li/debian/ ${DISTVERSION} main contrib non-free
 deb-src http://the.earth.li/debian/ ${DISTVERSION} main contrib non-free
 
 # Security Updates:
-deb http://security.debian.org/ ${DISTVERSION}/updates main contrib non-free
-deb-src http://security.debian.org/ ${DISTVERSION}/updates main contrib non-free
+deb http://security.debian.org/ ${SECURITY} main contrib non-free
+deb-src http://security.debian.org/ ${SECURITY} main contrib non-free
```
Comment (gbp) [inline bin/install-site.sh:170]: Either appears to work.

@sagepe should we add this path as it matches https://wiki.debian.org/DebianBullseye#FAQ It seems to be compatible Stretch and Buster too?

```suggestion
deb http://security.debian.org/debian-security ${SECURITY} main contrib non-free
deb-src http://security.debian.org/debian-security ${SECURITY} main contrib non-free
```

Comment (sagepe) [reply bin/install-site.sh:170]: Yes, why not, let's do that.

Comment (gbp) [reply bin/install-site.sh]: Done

**Review by sagepe — APPROVED** [review]

**Pull request merged**